packages/legend-art/style/base/_common.scss

```diff
@@ -108,6 +108,30 @@
   }
 }
 
+.dnd__entry-potential__dropzone__indicator {
```

Comment:
some of these styles are fine to be here, some of them better off be in `query buidler`, for example, specializations like `--full`, `--tall`, etc.